packages/datagateway-common/src/api/lucene.tsx

```diff
@@ -34,27 +34,35 @@ const urlParamsBuilder = (
     target: datasearchtype,
   };
 
-  const stringStartDate =
-    params.startDate !== null
-      ? format(params.startDate, 'yyyy-MM-dd')
-      : '00000-01-01';
-  const stringStartDateArray = stringStartDate.split('-');
-  query.lower =
-    stringStartDateArray[0] +
-    stringStartDateArray[1] +
-    stringStartDateArray[2] +
-    '0000';
+  //Do not assign query.lower for datafiles when there is no start date
+  //as this fixes issue when using lucene 4.10.0 instead of 4.11.1
+  if (datasearchtype !== 'Datafile' || params.startDate !== null) {
```

Comment:
Is it not possible to just have some logic that if both `startDate` and `endDate` are null then to not include them in the query? That feels more appropriate and would be something we wouldn't need to remove once we upgrade to 4.11.1

i.e. I feel just wrapping the original code in a `if (params.startDate !== null || params.endDate !== null)` would work and be future proof